src/main/java/seedu/address/model/score/Title.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.model.score;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the score title in a score.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTitle(String)}
+ */
+public class Title {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Title should only contain alphanumeric characters and spaces, and it should not be blank";
+
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```

Comment:
May I check that when you test the application, the same prefix will work fine?

Reply:
That will depend on how the ```AddScoreCommandParser``` is implemented.

Reply:
Because I was thinking if we have a different prefix maybe a bit easier

Reply:
@QQH0828 wdym by a different prefix